meta/FSMathLink.m

```diff
@@ -349,6 +448,26 @@
             "}\n"
           ];
 
+
+PutEffCouplings[modelName_] :=
+    Module[{prototype = "", body = "", function = ""},
+           prototype = "void put_eff_couplings(MLINK link) const;\n";
+
+           body = "check_spectrum_pointer();\n" <>
+                  "auto& effc = get_normalized_higgs_effc();\n\n" <>
```

Comment:
Can this local reference be made `const`?

Reply:
Done in 144bb81de15d96fdcd6236f50be31aadc0ae1bf2.